### ISSUE TITLE
Error parsing Angular.js expressions inside brackets

### ISSUE THREAD

**Issue by matfiz** [cognitom/gulp-slim#18]
Hi, 

I am trying to use gulp-slim in Angular.js app. I have problem with some Angular expressions inside brackets ().

Taking the working example, this:

````slim
small ({{248.12 | currency}})
````
would not be properly parsed by slim parser (and should result in <small>(248.12USD)</small>). The only solution I have found is to escape expressions like this with apostrophe ' like:

````slim
small '({{248.12 | currency}})
````
I suppose this maybe be fixed by providing Custom Parser Extension (like https://github.com/slim-template/sliq/blob/master/lib/sliq.rb), but I did not manage to make it working. Help will be appreciated.

**Comment by bitencode** [comment]
@matfiz - why are you putting your angular expression in side parenthesis `()`?  Slim is taking your `()` as a set of attributes on the html element - there is nothing @cognitom can do about that - it's the way Slim works.  We/I have tons of Slim markup with Angular, I have never had to put `{{...}}` inside `()`.  
The only alternative to not using the `()` I can think of is to force Slim to not see them as attribute delimiters by putting a blank set of attribute delimiters directly after your html element.

**Comment by matfiz** [comment]
@bitencode - thank you for clarification, you are absolutely right that Slim is taking string inside `()` as set of attributes and that was causing the error! As simple as it is, shame on me I did not see it myself!

Just to clarify: I also have tons of Slim markup with Angular and it works great. Only in this particular case I put the Angular code inside brackets just because I needed to have it formatted like that: `(248.12USD)`, no mysterious magic here ;-)

I close the issue.
